app/reducers/tabs.js

```diff
@@ -73,14 +75,6 @@ const addTab = ( state, tab ) =>
 
     let newState = [...state];
 
-    // Prevent http tabs at all
-    // TODO. This via middleware
-    if ( tab.url.startsWith( 'http' ) )
```

Comment:
how is this happening now?

Reply:
@joshuef This piece is already currently handled by [blockNonSafeRequests](https://github.com/maidsafe/safe_browser/blob/peruse/app/extensions/safe/blockNonSafeReqs.js#L27). Removing this allows just one external window to be opened instead of 3 to 4 for per click.

Reply:
👍